thoth/common/openshift.py

```diff
@@ -152,8 +152,8 @@ def __init__(
             _LOGGER.info(
                 "Using Argo Workflow to run jobs"
             )
-            import workflows
-            self.workflow_manager = workflows.WorkflowManager(
+            from .workflows import WorkflowManager
+            self.workflow_manager = WorkflowManager(
                 ocp_config={
```

Comment:
See reusing of `ocp_client` - https://github.com/thoth-station/common/pull/605 :ghost:

Reply:
I ll approve yours :)